extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/destination/MSQSelectDestination.java

```diff
@@ -27,21 +29,38 @@ public enum MSQSelectDestination
   /**
    * Writes all the results directly to the report.
    */
-  TASK_REPORT(false),
+  TASKREPORT("taskReport", false),
```

Comment:
We should use a snake case for the static variable names. Please revert it to the original values. 
Since this is changed for equality, we can create a static method that checks if the String provided equals the enum destination. 

```suggestion
  TASK_REPORT("taskReport", false),
```

Reply:
This was done so that we are able to still use the query context enum methods. Also I think `ResultFormat` class also uses a similar trick. Checkout `objectlines`

Reply:
Cool!